ecosystem/nft-metadata-crawler-parser/migrations/2024-01-31-221845_add_not_parsable_column/up.sql

```diff
@@ -1 +1 @@
-ALTER TABLE nft_metadata_crawler.parsed_asset_uris ADD COLUMN do_not_parse BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE IF NOT EXISTS nft_metadata_crawler.parsed_asset_uris ADD COLUMN do_not_parse BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
i think we try to make new columns nullable so that during migration it doesn't need to set every row to a value

Reply:
Didn't make nullable bc wanted it to auto set the value to false for existing columns 😄